shared/constants/crypto.tsx

```diff
@@ -81,11 +96,22 @@ export const Operations: {[key: string]: Types.Operations} = {
   Verify: 'verify',
 }
 
+const operationToInfoMessage: {[k in Types.Operations]: string} = {
```

Comment:
🐑 we've been making these types of things just `Map`s

Reply:
Thanks, good improvement